code/services/LDAPService.php

```diff
@@ -592,6 +616,205 @@ public function updateGroupFromLDAP(Group $group, $data)
     }
 
     /**
+     * Creates a new LDAP user from the passed Member record.
+     *
+     * @param Member $member
+     */
+    public function createLDAPUser(Member $member)
+    {
+        if (!$this->enabled()) {
+            return;
+        }
+
+        // Normalise username to lowercase to ensure we don't have duplicates of different cases
+        $member->Username = strtolower($member->Username);
```

Comment:
Do we need to check for empty Username here? Or document that createLDAPUSer expects that Member has a non null Username.